test/gtest/Makefile.am

```diff
@@ -160,16 +160,21 @@ gtest_SOURCES = \
 if HAVE_IB
 gtest_SOURCES += \
 	uct/ib/test_ib.cc \
-	uct/ib/test_ib_md.cc \
 	uct/ib/test_cq_moderation.cc \
 	uct/ib/test_ib_xfer.cc
 gtest_CPPFLAGS += \
 	$(IBVERBS_CPPFLAGS)
 gtest_LDADD += \
 	$(IBVERBS_LDFLAGS) \
 	$(top_builddir)/src/uct/ib/libuct_ib.la
+if HAVE_DEVX
 gtest_SOURCES += \
 	uct/ib/test_devx.cc
+endif
+if HAVE_MLX5_HW
+gtest_SOURCES += \
+    uct/ib/test_ib_md.cc
```

Comment:
minor: indentation should use tab instead of 4 spaces

Reply:
:+1:  I'll fix this in my next PR.